blocks/components/editable/index.js

```diff
@@ -233,7 +233,7 @@ export default class Editable extends wp.element.Component {
 		}
 
 		content = wp.element.renderToString( content );
-		this.editor.setContent( content );
+		this.editor.setContent( content, { format: 'raw' } );
```

Comment:
Re performance: I don't have a scenario in mind where we would be setting a lot.